firefox-ios/Client/Coordinators/Browser/BrowserCoordinator.swift

```diff
@@ -1020,7 +1020,7 @@ class BrowserCoordinator: BaseCoordinator,
             store.dispatch(
                 TabTrayAction(
                     windowUUID: self.windowUUID,
-                    actionType: TabTrayActionType.modalSwipedToClose
+                    actionType: TabTrayActionType.onViewDismissed
```

Comment:
This is called each time that the tab tray is dismissed:
- tab tray swiped down
- tab tray done button tapped
- automatically closed via close all tabs
- automatically closed via selected tab
- automatically closing one tab when having only one tab

Reply:
It is called when the navigationController has its `viewDidDisappear` called.